test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+import pytest
+from sklearn.metrics import recall_score, accuracy_score, confusion_matrix
+from fairlearn.experimental.enable_metric_frame_plotting import plot_metric_frame
+import matplotlib
+
+from fairlearn.metrics import MetricFrame
+from fairlearn.metrics._plotter import (
+    _GIVEN_BOTH_ERROR_BARS_AND_CONF_INT_ERROR,
+    _METRIC_AND_ERROR_BARS_NOT_SAME_LENGTH_ERROR,
+    _METRIC_AND_CONF_INTERVALS_NOT_SAME_LENGTH_ERROR,
+    _ERROR_BARS_MUST_BE_TUPLE,
+    _CONF_INTERVALS_MUST_BE_TUPLE,
+    _ERROR_BARS_NEGATIVE_VALUE_ERROR,
+    _CONF_INTERVALS_FLIPPED_BOUNDS_ERROR,
+)
+
+from .data_for_test import y_t, y_p, g_1
+
+z_score = 1.959964
+digits_of_precision = 4
+
+
+def wilson(p, n, digits=digits_of_precision, z=z_score):
+    """Return lower and upper bound."""
+    denominator = 1 + z**2/n
+    centre_adjusted_probability = p + z*z / (2*n)
+    adjusted_standard_deviation = np.sqrt((p*(1 - p) + z*z / (4*n)))/np.sqrt(n)
+    lower_bound = (centre_adjusted_probability - z*adjusted_standard_deviation) / denominator
+    upper_bound = (centre_adjusted_probability + z*adjusted_standard_deviation) / denominator
+    return (round(lower_bound, digits), round(upper_bound, digits))
+
+
+def compute_error_metric(metric_value, sample_size, z_score):
+    """Compute Standard Error Calculation (for Binary Classification).
```

Comment:
Looks like we're redefining a whole bunch of code here that was already in the example. Would it make sense to add these to the experimental module until we can get them merged into scipy or another package?

Reply:
I'm a bit ambivalent about that. On the one hand, it would be nice to eliminate the duplication of code. On the other.... if we're trying to avoid supporting it (because these error calculations aren't really our concern), then having it written out in the example notebook is a good thing I think.

Reply:
I could see us support it until we manage to contribute to another package and can replace it with that. Not supporting and then duplicating it all over the place (while at the same time not trying to contribute it elsewhere) seems weird to me. By "support" I mean even something like `experimental`. 

Also, if I remember correctly @hildeweerts mentioned that some of these are available in other packages. Why don't we use them from there?

Reply:
Hmmmm
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats._result_classes.BinomTestResult.proportion_ci.html?highlight=wilson
Maybe? We do have scipy as a dependency so it would be worth checking if this can be used instead.

I expect that there would still be wrapper code around it, though.... so yes, maybe that should go in an experimental module.

Reply:
That said, do we really need 'proper' error calculations in the tests? These are about plotting things given a bunch of numbers. So I think it would be acceptable to replace `recall_wilson` etc with something like
```python
def simple_error_bounds(y_true, y_pred):
   return (0.1, 0.2)
```
and similarly for the others.

Reply:
Good point. I guess since @alexquach already went through this trouble we can keep it as is. There is a case to be made for having tests with realistic values and these are definitely realistic since we're using them in the `examples` file.